js/app.js

```diff
@@ -0,0 +1,99 @@
+/*
+The user will input two numbers, a start and end value.
+When the user clicks "submit" or hits enter (13), the values
+trigger a function that adds all the numbers between them
+to the page in a list. The numbers divisible by 3 read "Fizz",
+5 "Buzz", and 15 "FizzBuzz".
+
+What we will have is a class named FizzBuzz with some methods
+inside of it. First is a function that gets the values of the
+input fields when the user clicks submit or pushes enter.
+
+The second will take those values and write all the nubmers
+inbetween to the DOM except those divisible by 3, 5, 15 each
+get Fizz, Buzz, or FizzBuzz, respectively.
+*/
+
+// IFFE around the JS Factory Pattern
+var FizzBuzz = (function () {
+
+  // Initialize the class
+  var _fizzBuzz = function () {
+
+    // Set up a variable to store the input values
+    var valArray;
+
+  };
+
+  _fizzBuzz.prototype = { // define fizzBuzz methods
+
+    // READING METHOD
+    // Store the starting and ending values in variables
+    getValues: function (startVal, endVal) {
+      this.valArray = [startVal, endVal];
```

Comment:
`this.valArray` is not the same `valArray` as on line 24. `valArray` from line 24 is unnecessary.